Gemfile.lock

```diff
@@ -423,3 +423,6 @@ DEPENDENCIES
   unicorn
   validates_timeliness (~> 3.0.14)
   will_paginate
+
+BUNDLED WITH
+   1.10.6
```

Comment:
Just tested and those lines have been removed from mine after runnning bundle from this branch. Any idea?